models/stg_hubspot__ticket.yml

```diff
@@ -158,3 +136,7 @@ models:
         description: Reference to the object type.
       - name: ticket_pipeline_id
         description: Reference to the pipeline.
+      - name: ticket_pipeline_stage_created_at
+        description: A timestamp representing when the record was created.
+      - name: ticket_pipeline_stage_updated_at
+        description: A timestamp representing when the record was updated.
```

Comment:
looks like the ticket_pipeline_stage and ticket_pipeline fields got mixed up in here

Reply:
Updated!